### DISCUSSION TITLE
Icon Request: fa-metronome

### DISCUSSION THREAD

**Discussion by simondoescode** [FortAwesome/Font-Awesome#21171] [Icon Requests]
![](http://i.imgur.com/dsa1cDk.png)

**Comment by mautay** [comment]
![logic-metronome](https://cloud.githubusercontent.com/assets/11761033/7948619/5dbe3976-093a-11e5-9346-5d2425888272.png)

Please create a metronome icon :)

**Comment by maxsennin** [comment]
So, i'd like a fa-metronome too.

Thx <3

**Comment by ColdVolcano** [comment]
👍 
![bpm](https://cloud.githubusercontent.com/assets/16726733/21363360/d8e1f88c-c6b1-11e6-9202-21bc75031590.png)

**Comment by AgentSmith0** [comment]
Any updates to this?

**Comment by hitnmix** [comment]
I'd like this too

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
